src/components/Contacts/ContactsTable/ContactsTable.tsx

```diff
@@ -26,12 +26,14 @@ export const ContactsTable: React.FC<Props> = ({
   filterPanelOpen,
   toggleFilterPanel,
 }: Props) => {
-  const { data, loading, error, refetch } = useContactsQuery({
-    variables: { accountListId },
+  const [searchTerm, setSearchTerm] = useState<string | undefined>(undefined);
```

Comment:
I think TS is happy with this which is slightly shorter. But same thing 🤷‍♂️ 
```suggestion
  const [searchTerm, setSearchTerm] = useState<string>();
```